### ISSUE TITLE
ERROR: resource not found: Settings

### ISSUE THREAD

**Issue by advance512** [convox/rack#2460]
Hi there,

Following a `convox instances keyroll`, and a standard app deploy a few days later, I started getting the error:
`ERROR: resource not found: Settings`
in response to most operaitons, including `deploy`, `scale` and `env`.

When I do `convox ps`, I only see around 1/3rd of the containers I expect to see - but all of them run and our app is accessible.

In the Rack stack of CloudFormation, I can see:

```
2018-01-22T17:54:20Z system:20180112161435/aws/cloudformation UPDATE_IN_PROGRESS porky User Initiated
2018-01-22T17:54:20Z service/web:20180112161435/bd5d6a005da0 ns=provider.aws at=SystemGet state=success elapsed=56.021
2018-01-22T17:54:20Z service/web:20180112161435/bd5d6a005da0 ns=api.controllers at=system.update state=success elapsed=1278.293
2018-01-22T17:54:20Z service/web:20180112161435/bd5d6a005da0 ns=api.web at=request state=success status=200 method="PUT" path="/system" elapsed=1278.486
2018-01-22T17:54:21Z service/web:20180112161435/f40df05bb1e0 ns=api.web at=request state=success status=200 method="GET" path="/check" elapsed=0.109
2018-01-22T17:54:23Z system:20180112161435/aws/cloudformation UPDATE_ROLLBACK_IN_PROGRESS porky Export porky:DomainInternal cannot be deleted as it is in use by porky-selectom
2018-01-22T17:54:24Z service/web:20180112161435/bd5d6a005da0 ns=api.web at=request state=success status=200 method="GET" path="/check" elapsed=0.108
``` 

I checked the Settings output of the stack, and it points to an S3 bucket that is accessible (to my user).

Any idea what this is, or how I can fix it?

**Comment by advance512** [comment]
After recreating the rack from scratch, creating a new app, and deploying it - I still face the issue.

So, it is reproducible. And it is a killer, as it killed our production.

**Comment by ddollar** [comment]
Solved in #2463
